src/figma/tokens.json

```diff
@@ -1177,5 +1177,6 @@
         }
       }
     }
-  }
+  },
+  "$themes": []
```

Comment:
@Akatori-Design Did you create a "themes" category? Wondering if this is just an updated from the plugin itself?

Reply:
Added by figma tokens